go.mod

```
@@ -3,7 +3,7 @@ module github.com/kopia/kopia
 go 1.22.7
```

```suggestion
go 1.22
```
This PR relaxes go version required via https://github.com/grpc/grpc-go/releases/tag/v1.68.1

As such, and also to close #4292, this PR should be updated or followed up by to remove the patch go version.